docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -2062,6 +2093,102 @@ Override the default architecture variant of the container image.
 
 This is equivalent to the Podman `--variant` option.
 
+## Login units [Login]
+
+Login files are named with a `.login` extension and contain a section `[Login]` describing the
+login command. The generated service is a one-time command that logs into a registry.
+
+Using login units allows pulling images from private registries without having to login manually.
+
+There is only one required key, `Registry`, which defines the URL of the registry to log into.
+
+To avoid password leaking, Quadlet will set the permissions of the generated service file as 0600.
+It is recommended to do the same for `.login` unit files.
+
+Valid options for `[Login]` are listed below:
+
+| **[Login] options**                             | **podman login**                                                     |
+|-------------------------------------------------|----------------------------------------------------------------------|
+| AuthFile=/etc/registry/auth\.json               | --authfile=/etc/registry/auth\.json                                  |
+| CertDir=/etc/certs                              | --cert-dir=/etc/certs                                                |
+| LogoutOnStop=true                               | Add ExecStopPost to log out of the registry when the unit is stopped |
+| Password=mypassword                             | --password-stdin and set StandardInputText                           |
+| PodmanArgs=--compat-auth-file=/etc/compat\.json | --compat-auth-file=/etc/registry/compat\.json                        |
+| Registry=quai.io                                | podman login quai.io                                                 |
+| Secret=mysecret                                 | --secret=mysecret                                                    |
+| ServiceName=name                                | Name the systemd unit `name.service`                                 |
+| TLSVerify=false                                 | --tls-verify=false                                                   |
+| Username=myuser                                 | --username=myuser                                                    |
+
+### `AuthFile=`
+
+Path of the authentication file.
+
+This field is mandatory when linking between an `AuthFile` key of a different unit and the `.login` unit.
+
+This is equivalent to the `--authfile` option.
```

Comment:
```suggestion
This is equivalent to the Podman `--authfile` option.
```

In some places, you have `Podman` for a line like this, in others, you do not.  I prefer having Podman for each.  However, your call, I'd just ask for consistency.